### PR TITLE
season 11/12 updates

### PR DESCRIPTION
**Pull request by delphiactual**
This PR will remain open until all season 11 info is revealed.

### CODE REVIEW

**Comment by delphiactual** [comment]
Reminder: Remove all references to`D2CurrentSeason` in DIM either prior to merge or in the PR to DIM.

File: data/seasons/d2-season-info.ts [hunk 1]
```diff
@@ -10,11 +10,13 @@ export enum D2SeasonEnum {
...
   WORTHY,
+  REDACTED_11, // TODO: Update on verification
+  REDACTED_12, // TODO: Update on verification
 
-  __LENGTH, // This always needs to be last
+  __LENGTH__, // This always needs to be last
 }
 
-export const D2CurrentSeason: number = D2SeasonEnum.__LENGTH - 1;
+export const D2CurrentSeason: number = D2SeasonEnum.__LENGTH__ - 1;
```
Comment (thepoofy) [inline data/seasons/d2-season-info.ts]: If you're adding seasons 11 and 12, will this be off by one?

Comment (delphiactual) [reply data/seasons/d2-season-info.ts]: It is an unused variable, hence the linked PR that has been merged

**Comment by delphiactual** [comment]
Closed by #144

**Pull request closed without merging**